todolist/app/screens/App/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes  from 'prop-types';
+import { View, StyleSheet } from 'react-native';
+import { connect } from 'react-redux';
+
+import Title from '../../components/Title';
+import Footer from '../../components/Footer';
+import Input from '../../components/Input';
+import List from '../../components/List';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+})
+
+class App extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Title />
+        <Input />
+        <List items={this.props.items} />
```

Comment:
i think it's best to manage the items inside the List component. You are not using the items for anything here in the App component. So you can mapStateToProps like here but in the List's index